doc/ansible/user-guide.md

```diff
@@ -201,15 +201,16 @@ Modify `roles/memcached/tasks/main.yml` to look like the following:
 ```yaml
 ---
 - name: start memcached
-  community.kubernetes.k8s:
+  k8s:
+    state: present
```

Comment:
You replaced it because you face: 

 ERROR! couldn't resolve module/action 'community.kubernetes.k8s'. This often indicates a misspelling, missing collection, or incorrect module path.                                                                                                      │
│ The error appears to be in '/opt/ansible/roles/memcached/tasks/main.yml': line 2, column 3, but may                                                                                                                                                      │
│ be elsewhere in the file depending on the exact syntax problem.                                                                                                                                                                                          │
│ The offending line appears to be:                                                                                                                                                                                                                        │
│ ---                                                                                                                                                                                                                                                      │
│ - name: start memcached                                                                                                                                                                                                                                  │
│   ^ here

However, it is the master branch and not the latest tag release. Did you check it with the master branch?
See that to use the master branch your Dockerfile should be as follows:

```
FROM quay.io/operator-framework/ansible-operator:master

COPY watches.yaml ${HOME}/watches.yaml
COPY roles/ ${HOME}/roles
```

And you need to update your project with all break changes.  See; https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#breaking-changes-2

Reply:
Note that it was changed for the changes made in the master and the doc of the latest release is as you sugessted here; https://github.com/operator-framework/operator-sdk/blob/v0.15.x/doc/ansible/user-guide.md

Reply:
Ah, I have used the version installed through homebrew which is also the latest version.

```
~ $ operator-sdk version
operator-sdk version: "v0.15.2", commit: "ffaf278993c8fcb00c6f527c9f20091eb8dd3352", go version: "go1.13.8 darwin/amd64"
~ $
```